src/js/actions/export.js

```diff
@@ -439,10 +448,21 @@ define(function (require, exports) {
     setAllNonABLayersExportEnabled.transfers = [setLayerExportEnabled];
 
     /**
+     * Open a dialog to allow the user to choose a directory
+     * @return {Promise.<string>} Resolves with pathName, or rejects if user-canceled
+     */
+    var promptForFolder = function () {
```

Comment:
derp.  I think I originally was orchestrating this from outside the action (JSX), but now I'm just calling _exportService directly in `exportAllAssets`.  I'll remove this as I don't see any reason we would want to transfer to it right now.